### ISSUE TITLE
windows python error

### ISSUE THREAD

**Issue by scottdraves** [twosigma/beakerx#2352]
from the 2nd example in the python tutorial in windows:

```
C:\Users\Beaker\beaker-notebook\core\config\plugins\eval\ipythonPlugins\ipython\beaker_runtime.py in default(self, obj)
    309             out = {}
    310             out['type'] = "Date"
--> 311             out['timestamp'] = int(obj.strftime("%s"))*1000
    312             return out
    313         if type(obj) == pandas.core.frame.DataFrame:

ValueError: Invalid format string
```

**Comment by scottdraves** [comment]
http://stackoverflow.com/questions/10807164/python-time-formatting-different-in-windows
cap S?

**Comment by maxyeg** [comment]
It fixes #2292
